### PR TITLE
gegl_0_3: 0.3.26 → 0.3.28

### PR DESCRIPTION
**Pull request by jtojnar**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_3_0

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

**Comment by jtojnar** [comment]
@GrahamcOfBorg build gegl_0_3

File: pkgs/development/libraries/gegl/3.0.nix [hunk 1]
```diff
@@ -3,21 +3,27 @@
...
   configureFlags = "--disable-docs";
 
   enableParallelBuilding = true;
 
-  doCheck = true;
+  # GEGL_SWAP=RAM GEGL_PATH=../../operations ./buffer-test
+  # dyld: Library not loaded: @rpath/libjson-glib-1.0.0.dylib
+  #   Referenced from: /private/tmp/nix-build-gegl-0.3.28.drv-0/gegl-0.3.28/tests/buffer/.libs/buffer-test
+  #   Reason: image not found
+  doCheck = !stdenv.isDarwin;
```
Comment (LnL7) [inline pkgs/development/libraries/gegl/3.0.nix]: I think dependencies that link against this library will run into the same issue.

Comment (jtojnar) [reply pkgs/development/libraries/gegl/3.0.nix]: What could be causing it? We should probably fix it before GIMP 2.10 is released.

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Comment by jtojnar** [comment]
It does not yet build on Darwin but #34431 will fix it.

**Pull request merged**